packages/web-components/case-card/src/element.ts

```diff
@@ -0,0 +1,150 @@
+import css from './css.js';
+
+export interface CardData {
+  linkAriaLabel: string;
```

Comment:
```suggestion
  linkLabel: string;
```

aria is een implementatie-detail